src/citra_qt/main.cpp

```diff
@@ -499,6 +499,21 @@ void GMainWindow::RestoreUIState() {
     statusBar()->setVisible(ui.action_Show_Status_Bar->isChecked());
 }
 
+void GMainWindow::OnAppFocusStateChanged(Qt::ApplicationState state) {
+    if (UISettings::values.pause_when_in_background) {
```

Comment:
Inverting this into a guard clause like:

```suggestion
    if (!UISettings::values.pause_when_in_background) {
        return;
    }
```

will allow you to unindent code one level